ansible/roles/cyhy_mailer/tasks/main.yml

```diff
@@ -41,13 +41,13 @@
       region = {{ ses_aws_region }}
       role_arn = {{ ses_send_email_role }}
 
-# docker-compose will automatically use docker-compose.yml and
+# docker compose will automatically use docker-compose.yml and
 # docker-compose.override.yml, so this is a way for us to tune
-# docker-compose's behavior to the particular machine.
+# docker compose's behavior to the particular machine.
```

Comment:
Continuation since GitHub won't let me include the unmodified line in a complete suggestion.

```suggestion
# compose's behavior to the particular machine.
```